test/callflows/callflows_service_test.dart

```diff
@@ -1,20 +1,20 @@
-import 'dart:convert';
 import 'dart:io';
 
 import 'package:messagebird/voice_calling.dart';
 import 'package:test/test.dart';
 
+import '../credentials.dart';
+
 void main() {
+  final Credentials credentials = Credentials.from(Platform.environment);
+
   group('CallflowService', () {
-    Map credentials;
     CallflowsService callflowsService;
     Callflow callflow;
     String id;
 
     setUp(() {
-      credentials =
-          json.decode(File('test_resources/keys.json').readAsStringSync());
-      callflowsService = ApiCallflowsService(credentials['live']);
+      callflowsService = ApiCallflowsService(credentials.API_LIVE_KEY);
```

Comment:
Do you know if its possible to use the TEST keys here instead?

Reply:
I don't know by heart, but in my experience what you can do with the test key is very limited. It mainly has the purpose of verifying that the connection has been established.

Reply:
Nevermind! I was wrong, new commits incoming where the callflow service uses the test key